### PR TITLE
Rename the exempt status of PIA and STRA

### LINKED ISSUE #65: Remove Exempt status option
Replace "Exempt" with "Not Required - Reviewed with MISO". Description can be the same, with the addition that "the Ministry Information Security Officer has been consulted and agrees that no STRA is required for this repo."

This change is due to terminology used for the Exemption process, which is different from what is taking place here.

### PR DESCRIPTION
**Pull request by jleach**
As requested, this PR renames the `exempt` status for PIA and STRA to `not-applicable`.

Fixes #65

### CODE REVIEW

File: __tests__/repository.test.ts [hunk 1]
```diff
@@ -107,4 +125,90 @@ describe('Repository management', () => {
...
+        expect(fetchFile).toBeCalled();
+        expect(loadTemplate).toBeCalled();
+        expect(addFileViaPullRequest).toBeCalled();
+    });
+
+    it('Valid compliance status should not be updated', async () => {
+        const owner = 'bcgov';
+        const repo = 'hello5';
+
+        doc.spec.forEach(s => {
```
Comment (patricksimonian) [inline __tests__/repository.test.ts:161]: just a personal opinion but its better to not mutate the original doc spec, could lead to bugs in other tests. 

```js
const completedDocs = {
...doc,
spec: doc.spec.map(s => ({...s, status: 'completed' }))
}
```

File: templates/why-rename-status.md [hunk 2]
```diff
@@ -0,0 +1,3 @@
+We need to rename the status `exempt` to `not-requried` because the word "exempt" is used for other purposes within a STRA or PIA review process. To avoid confusion, and make this as simple as possible, I've created this pull request with the required change.
```
Comment (patricksimonian) [inline templates/why-rename-status.md:1]: not-required

File: __tests__/repository.test.ts (hunk 1 above)
Comment (jleach) [reply __tests__/repository.test.ts:161]: Good call. I usually do this. I'll go back and fix it.

File: templates/why-rename-status.md (hunk 2 above)
Comment (jleach) [reply templates/why-rename-status.md:1]: Fixed.

**Pull request merged**